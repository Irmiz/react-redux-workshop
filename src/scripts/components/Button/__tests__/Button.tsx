import * as React from 'react';
import Button from './../../Button';
import renderer = require('react-test-renderer');

describe('Button', () => {
    it('renders a button', () => {
        const buttonComponent = renderer.create(
            <Button />
        ).toJSON();

        expect(buttonComponent).toMatchSnapshot();
    });
});
