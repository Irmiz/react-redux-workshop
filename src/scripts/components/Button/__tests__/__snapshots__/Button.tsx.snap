exports[`Button renders a button 1`] = `
<button
  type="button" />
`;
